### PR TITLE
add ability to keep conda env, add removal of repos dir, first hack a…

### PR DESCRIPTION
**Pull request by kmarsteller**
…t readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+"python benchmark.py openmdao" to benchmark using "openmdao.json"
+results will be kept in a similarly-named database, e.g. "openmdao.db"
+commands and their output will be kept in a similarly-named log, e.g "openmdao.log"
+
+
+OPTIONS:
+________
+
+To plot a specific spec from the database, after benchmarking has been run, use:
+"python benchmark.py --plot spec"
```
Comment (swryan) [inline README.md]: you need the project argument here, so it knows what database to pull the data from

Comment (kmarsteller) [inline README.md]: d'oh!

File: benchmark.py [hunk 2]
```diff
@@ -153,14 +152,14 @@ def cd(newdir):
...
-def repo(repository, branch=None):
+def repo(repository, name, branch=None):
     """
     cd into local copy of repository.  if the repository has not been
     cloned yet, then clone it to working directory first.
     """
     prev_dir = os.getcwd()
 
-    repo_dir = os.path.expanduser(conf["repo_dir"])
+    repo_dir= os.path.expanduser(os.path.join(conf["working_dir"], (name +"_repos")))
```
Comment (swryan) [inline benchmark.py]: this decorator is actually used for all the repos (project, trigger and dependency) so you don't want a separate repos_dir each time the decorator is used...  I think you want to determine the repos_dir name from the project name at the benchmark level and pass it down through the decorator

File: benchmark.py [hunk 3]
```diff
@@ -183,12 +182,15 @@ def repo(repository, branch=None):
...
 
-def benchmark(project_info, force=False):
+def benchmark(project_info, force=False, keep_env=False):
     current_commits = {}
     update_triggered_by = []
 
     db = BenchmarkDatabase(project_info["name"])
-
+    
+    #remove previous repos and clone fresh ones to avoid trouble.
```
Comment (swryan) [inline benchmark.py]: so here is where you probably want to figure out the repo_dirs name based on the project name, than pass that name to `remove_repos_dir` and the subsequenct `with repos`

**Pull request merged**